app/views/jobseekers/job_applications/build/personal_statement.html.slim

```diff
@@ -8,6 +8,18 @@
     h1.govuk-heading-l = t(".heading")
     p.govuk-body = t(".description")
 
+    - if job_application.vacancy.documents.any?
+      = govuk_notification_banner(title: t(".banner_title")) do |banner|
+        - banner.add_heading
+        = t(".banner_description_html", job_link: govuk_link_to(job_application.vacancy.job_title, job_path(job_application.vacancy), class: "document-link-component__link", target: "_blank"))
+        ul.hide-bullet-points
+          - job_application.vacancy.documents.in_groups_of(1, false).each do |row|
```

Comment:
In fact we might be able to just do 
```slim
= render Shared::DocumentLinkComponent.with_collection(job_application.vacancy.documents)
```
I'm not sure why I did it this other way before